java/arcs/sdk/android/storage/service/StorageService.kt

```diff
@@ -165,9 +170,31 @@ open class StorageService : ResurrectorService() {
         }
     }
 
+    override fun onUnbind(intent: Intent): Boolean {
+        if (intent.action == MANAGER_ACTION) {
+            return false
```

Comment:
Might be better to just return `super.onUnbind(intent)` here too, like with how it's done on line 191.

Reply:
Yeah, you're right.